05.ls/ls.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+COLUMN = 3
```

Comment:
```suggestion
COLUMN_LENGTH = 3
```

みたいにしておくとよりわかりやすいかもしれません